tests/ts_B_select_pass_3_sorted_on_key.erl

```diff
@@ -6,22 +6,25 @@
 
```

Comment:
No, it's pretty random.  I'm happy adding the boilerplate everywhere.  Depends upon the developer's mood, I think.  Currently very inconsistent.  Want me to update all tests?